homeassistant/components/reisingerbridge/__init__.py

```diff
@@ -0,0 +1,79 @@
+"""The Reisinger Bridge integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+import reisingerdrive
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import update_coordinator
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import CONF_DEVICE_KEY, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.COVER]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Reisinger Bridge from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+
+    open_reisinger_connection = reisingerdrive.OpenReisinger(
```

Comment:
Should you not use validate_conection here just as you do in config_flow? To make sure the connection is still working.